media-sound/beets/beets-1.4.9-r3.ebuild

```diff
@@ -26,7 +26,7 @@ HOMEPAGE="http://beets.io/ https://pypi.org/project/beets/"
 
```

Comment:
HOMEPAGE="http://beets.io/ ............... This could use https.

Reply:
@charIes17 
thank you for the feedback,
I upgraded the branch.